Polyfills/Window/Source/Window.cpp

```diff
@@ -8,20 +8,28 @@
 
 namespace
 {
-    struct TimeoutFunction
+    using TimeoutId = Babylon::Polyfills::Internal::TimeoutId;
+    using Milliseconds = std::chrono::milliseconds::rep;
+    using TimePoint = std::chrono::time_point<std::chrono::steady_clock, std::chrono::milliseconds>;
```

Comment:
I'm not sure I understand what is going on with this. It's a bit confusing to use `Milliseconds` as a `rep` which somewhat defeats the purpose of using `std::chrono::duration`.